### Hello! ðŸ‘‹

My name is Jiri Novotny, I come from the Czech Republic, and I am a [bioinformatician](https://en.wikipedia.org/wiki/Bioinformatics).

I am currently working for [Immunai](https://www.immunai.com) as a computational biologist / software engineer.

[![LinkedIn](https://user-images.githubusercontent.com/15315342/197699327-7c706080-3f9e-4cd6-872d-920911ab5540.png)](https://www.linkedin.com/in/ji%C5%99%C3%AD-novotn%C3%BD-1b6a25182/)
[![ORCID_iD](https://user-images.githubusercontent.com/15315342/197780012-18858dc8-03da-4492-a4e5-2172245d1606.png)](https://orcid.org/0000-0003-1338-638X)

### What do I do in bioinformatics? ðŸ¤“

I was doing Ph.D. at the [Institute of Molecular Genetics of the Czech Academy of Sciences](https://www.img.cas.cz/en/)
in the [Laboratory of Genomics and Bioinformatics](https://www.img.cas.cz/group/michal-kolar/).
There I was usually analyzing [transcriptomic](https://en.wikipedia.org/wiki/Transcriptome) data: bulk and single-cell RNA-seq, and qPCR.
But I also have some proficiency in general bioinformatics (all those formats, command line tools, etc.) and other methods, e.g. variant calling, chromatin accessibility (ATAC-seq), TCR sequencing, etc.

- See [scdrake](https://github.com/bioinfocz/scdrake) - a pipeline for droplet-based single-cell RNA-seq data.

Now at Immunai I am mostly researching and benchmarking new bioinformatic methods and making sure those we like will land in our internal pipelines (which I also sometimes write from scratch).
I am also giving support to our single-cell data curators and of course, fixing bugs (I know, very much unexpected ðŸ™ƒ).

### What can I do besides bioinformatics? ðŸ¤”

I am fluent in R and Python and have experience in making pipelines, general data wrangling and visualization, web development
(Python/Django, R/Shiny, HTML5/CSS3), UNIX scripting (bash) and system administration,
Git versioning and continuous integration (GitHub Actions). I like to write clean code and documentation
(and really appreciate if others also do so ðŸ˜Š). Occasionally, I contribute to open-source projects, or at least open constructive issue tickets.

- See [AmtDB](https://amtdb.org/) - database of ancient human mitochondrial DNA (private code, full-stack developer role).

I was also teaching practicals in the Analysis of Gene Expression at the [University of Chemistry and Technology in Prague](https://www.vscht.cz//?jazyk=en).

- See [exercise materials for the Analysis of Gene Expression](https://github.com/gorgitko/analysis_of_gene_expression) if you want to learn something about the analysis of RNA-seq etc.
  (these materials are well-suited for self-study)

To grab the chance to promote my study programme, here is the link: <http://studuj.bioinformatiku.cz/>

I would say, it's not an easy, but interesting and satisfying field and the future is bright as more people are needed to analyze
the neverending, large bunch of biological data or to develop tools that make this task easier for others.

Several times I was also participating as a lecturer at
[Prague Autumn School - RNASeq Data Analysis Workshop](https://www.seqme.eu/en/courses/dates/ngs-autumn-school) organized by
[SeqMe company](https://www.seqme.eu/en).

<!--
**gorgitko/gorgitko** is a âœ¨ _special_ âœ¨ repository because its `README.md` (this file) appears on your GitHub profile.

Here are some ideas to get you started:

- ðŸ”­ Iâ€™m currently working on ...
- ðŸŒ± Iâ€™m currently learning ...
- ðŸ‘¯ Iâ€™m looking to collaborate on ...
- ðŸ¤” Iâ€™m looking for help with ...
- ðŸ’¬ Ask me about ...
- ðŸ“« How to reach me: ...
- ðŸ˜„ Pronouns: ...
- âš¡ Fun fact: ...
-->
